webview/src/shared/dispatchActions.ts

```diff
@@ -0,0 +1,29 @@
+import { UnknownAction } from '@reduxjs/toolkit'
+import { TableData } from 'dvc/src/experiments/webview/contract'
+import { PlotsData } from 'dvc/src/plots/webview/contract'
+import { SetupData } from 'dvc/src/setup/webview/contract'
+import type { ExperimentsActions } from '../experiments/components/App'
+import type { ExperimentsDispatch } from '../experiments/store'
+import type { PlotsActions } from '../plots/components/App'
+import type { PlotsDispatch } from '../plots/store'
+import type { SetupActions } from '../setup/components/App'
+import type { SetupDispatch } from '../setup/store'
```

Comment:
[F] Using `type` here to try and avoid including unnecessary code in each webview's bundle.